### PR TITLE
Fix datefields' DefaultValue

### PR DESCRIPTION
**Pull request by herflis**

### CODE REVIEW

**Review by pusztaienike — APPROVED** [review]
🍹

File: apps/sensenet/src/components/field-controls/DatePicker.tsx [hunk 1]
```diff
@@ -39,11 +39,16 @@ const useStyles = makeStyles((theme: Theme) => {
...
+  const initialState =
+    props.fieldValue ||
+    changeTemplatedValue(props.settings.DefaultValue, props.settings.EvaluatedDefaultValue) ||
+    moment().toISOString()
   const [value, setValue] = useState(initialState)
   const classes = useStyles()
 
   const handleDateChange = (date: MaterialUiPickersDate) => {
+    console.log(initialState)
+    console.log(moment.utc(date).toISOString())
```
Comment (taki9) [inline apps/sensenet/src/components/field-controls/DatePicker.tsx:51]: remove console logs

File: packages/sn-controls-react/src/helpers.ts [hunk 2]
```diff
@@ -3,8 +3,16 @@ import { FieldSetting } from '@sensenet/default-content-types'
 /**
  * Search for '[Script:jScript]' tag in string and returns empty string when found
  */
-export const changeJScriptValue = (value: string | undefined) =>
-  value && (value.includes('[Script:jScript]') ? '' : value)
+
+export const changeTemplatedValue = (value: string | undefined, evaluatedValue?: string | undefined) => {
+  if (value && value.includes('@@')) {
```
Comment (taki9) [inline packages/sn-controls-react/src/helpers.ts:8]: value?.includes

File: packages/sn-controls-react/src/helpers.ts [hunk 3]
```diff
@@ -3,8 +3,16 @@ import { FieldSetting } from '@sensenet/default-content-types'
...
+    return evaluatedValue
+  } else if (value && value.includes('[Script:jScript]')) {
```
Comment (taki9) [inline packages/sn-controls-react/src/helpers.ts:10]: value?.includes

**Review by taki9 — CHANGES_REQUESTED** [review]

**Review by taki9 — APPROVED** [review]

**Pull request merged**